tools/build-tools/src/typeValidator/typeData.ts

```diff
@@ -74,13 +75,22 @@ function getNodeTypeData(node:Node, namespacePrefix?:string): TypeData[]{
             ? `${namespacePrefix}.${node.getName()}`
             : node.getName()!;
 
+        let typeParams: string | undefined;
+        if(Node.isInterfaceDeclaration(node) || Node.isTypeAliasDeclaration(node)){
+
+            if(node.getTypeParameters().length > 0){
+                typeParams = `<${node.getTypeParameters().map(()=>"any").join(",")}>`;
```

Comment:
it's really hard to build the right type for a generic, so for now we'll just pass any, as it will always work

Reply:
maybe worth adding as a comment in code too